content/persons/christophe-herrou.md

```diff
@@ -0,0 +1,84 @@
+---
+# Copie colle ce fichier pour créer ton profil sur PopCorn.
+# Le nom du fichier sera aussi l'url de ton profil
+
+# REQUIS
+# Le titre qui te refléte "Expert JavaScript", "Développeur web junior"
+titre: Développeur Front-end, UX designer
+
+# OPTIONNEL
+# Un sous-titre de ton choix si tu as quelque chose à ajouter
+sous_titre: Design fonctionnel, empathie, création et chouquettes !
+# REQUIS
+prenom: Christophe
+
+# REQUIS
+nom: Herrou
+
+# REQUIS
+# ATTENTION : cette donnée est PUBLIQUE et en CLAIR dans le dépot git et sur le site
+#
+# Le mail qui permet au client de contacter pour te faire une proposition de projet
+# et de t'inviter sur le slack de PopCorn.
+#
+# Si cela t'embête parce que tu utilises une adresse perso, tu peux aussi te créer
+# une adresse mail pro publique et la rediriger vers ta boîte mail perso
+mail: christophe.herrou@madeoweb.com
+
+# OPTIONNEL
+# ATTENTION cette donnée est PUBLIQUE et en CLAIRE dans le dépot git et sur le site
+telephone: 06 69 31 76 62
+
+# REQUIS
+# l'image doit être ajoutée dans le dossier /static/media/photos et faire moins de 100ko !
+photo: christophe-herrou.jpg
+
+# REQUIS
+# Les domaines métiers que tu exerces **vu par tes client(e)s sans connaissances
+# techniques**, pour qu'ils ou elles te trouvent facilement. Un client(e) veut par exemple
+# réaliser une des choses suivantes :
+# - sites web
+# - applications mobiles
+# - jeux vidéos
+# - boutiques en ligne
+# - site vitrine
+# - ... à toi de jouer
+domaines_metiers:
+  - sites web
+  - applications mobiles
+
+# REQUIS
+# Les technologies que tu utilises
+# Une technologie peut être une langage, un framework, une librairie, un CMS ...
+technologies:
+  - HTML/CSS
+  - Bootstrap
+  - JavaScript
+  - Wordpress
+  - Ruby
+  - Ruby on rails
+
+# OPTIONNEL
+# Les différents réseaux, liens utils à afficher sur ton profil
+# Ça peut être un réseau social, un site internet ...
+networks:
+  - name: Site Internet
+    value: https://madeoweb.com
+  - name: Linkedin
+    value: linkedin.com/in/christopheherroudéveloppeurfrontenduxdesignernantes
```

Comment:
Ton lien n'est pas bon à mon avis.

Reply:
Pardon j'ai du mal ciblé mon commentaire de PR, mais ça concernait surtout ton lien LinkedIn 😅

`linkedin.com/in/christopheherroudéveloppeurfrontenduxdesignernantes`